mdr/mdr.py

```diff
@@ -110,7 +143,11 @@ def score(self, features, classes):
             The estimated accuracy based on the constructed feature
 
         """
-        return 0.0
+        new_feature = self.fit_transform(features, classes)
+        results = (new_feature == classes).all()
+        score = np.sum(results)
+        accuracy_score = score/classes.size 
```

Comment:
In general, make sure that all divisions in this code involve `float` values. Integer division will not give us the correct value.